### DISCUSSION TITLE
Does this mod work with the RE8 DLC?

### DISCUSSION THREAD

**Discussion by netriosilver** [praydog/REFramework#619] [General Questions] [answered]
I'm having trouble finding out online whether the mod works with the new characters (Rose, Heisenberg, etc.) in the DLC. I've been waiting to buy the DLC till I could at least play Shadows of Rose in first-person VR. If not available now, is this planned?

**Comment by praydog** [comment, marked as answer]
1. No, third person only. 
2. Maybe. Not working on it, I am busy with Unreal.

**Reply by netriosilver** [reply]
Alright cool

**Reply by J4cKP0t777** [reply]
Any compatible First person mods out there? Shadows of Rose looks interesting but I cant go back to pancake mod :)
Thanks for ur work Praydog
